calico/maintenance/install-apiserver.md

```diff
@@ -132,6 +132,8 @@ After following the above steps, you should see the API server pod become ready,
    profiles                                                                          projectcalico.org              false        Profile
    ```
 
+If the above command returns `error: unable to retrieve the complete list of server APIs: projectcalico.org/v3: the server is currently unable to handle the request` you should check that the `kube-apiserver` is able to communicate with the `calico-apiserver` on TCP 5443.
```

Comment:
nit: I think that since this is a troubleshooting step, it might be good to put in a `**Note:** {: .alert .alert-info}` section

Reply:
I'm not sure it is appropriate at all here to be honest, it might be better placed in our troubleshooting documentation rather than this section. 

The install page isn't really the right location for arbitrary debugging information. Rather, we should make sure any networking requirements are laid out in the requirements and prerequisites documentation.